.ci/install_nemu.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+cidir=$(dirname "$0")
+source "${cidir}/lib.sh"
+source /etc/os-release || source /usr/lib/os-release
+
+arch=$("${cidir}"/kata-arch.sh -d)
+
+clone_build_install() {
+	local nemu_repo=https://github.com/intel/nemu.git
+	local nemu_repo_dir="${HOME}/nemu"
+	local nemu_build_dir=
+	local nemu_bin=
+	git clone "${nemu_repo}" "${nemu_repo_dir}"
+	pushd "${nemu_repo_dir}"
+	git fetch origin
+	git checkout experiment/automatic-removal
+
+	case "$arch" in 
+	x86_64)
+		nemu_build_dir="${HOME}/build-${arch}_virt/${arch}_virt-softmmu/"
+		nemu_bin="qemu-system-${arch}_virt"
+		SRCDIR="$PWD" ./tools/build_x86_64_virt.sh
+		;;
+	aarch64)
+		nemu_build_dir="${HOME}/build-${arch}/${arch}-softmmu/"
+		nemu_bin="qemu-system-${arch}"
+		SRCDIR="$PWD" ./tools/build_aarch64.sh
+		;;
+	*)
+		die "Unsupported architecture: $arch"
+		;;
+	esac
+	popd
+	rm -rf "${nemu_repo_dir}"
+}
+
+install_firmware() {
+	local firmware="OVMF.fd"
+	# Currently there are changes in the firmware that are not upstream,
+	# so we need to take it from this repository.
+	local firmware_url="https://github.com/rbradford/edk2/releases/download/0.3/${firmware}"
```

Comment:
Thought: this could be added to https://github.com/kata-containers/tests/blob/master/versions.yaml ?

Reply:
agree... will add it here